src/main/java/cz/muni/fi/timeline/service/TimelineCommentService.java

```diff
@@ -1,7 +1,46 @@
 package cz.muni.fi.timeline.service;
 
+import cz.muni.fi.timeline.entity.TimelineComment;
+
+import java.util.List;
+import java.util.Optional;
+
+
+/**
+ * Service for manipulation with timeline comments.
+ *
+ * @author Ondřej Kuhejda
+ */
 public interface TimelineCommentService {
-    // add comment to timeline
```

Comment:
Missing function to add comment to timeline

Reply:
Já bych to vytváření dělal tak, že bych do commentu přidal odkaz na timelinu (nový field). Tím by stačilo volat jen create pomocí DAO. A dalo by se to NotNull aby každý comment musel mít nastavenou timelinu při vytváření. Co si o tom myslíš?

Reply:
Podľa mňa to je OK tak ako to je teraz. Stačí ak vytvorí raz Comment pri pridaní do Timeline. Nevidím benefity keby bol v Timeline field v Comment.

Reply:
Ten benefit bude v tom, že zaručíme, že každý komentář musí být přiřazen k nějaké timelině (nemůže existovat sám o sobě).

Reply:
Môžeme to urobiť tak, že Comment sa bude vytvárať len keď sa pridáva do Timeline nový Comment (napr. v nejakej Facade). Nevidím iný use case, kedy by sa vytváral Comment, takže týmto zaručíme, že nebude nikdy existovať bez Timeline. Aspoň sa vyhneme ďalšiemu fieldu.